graspy/align/seedless_procrustes.py

```diff
@@ -0,0 +1,328 @@
+    transport problem via Sinkhorn algorithm, whereas obtaining an orthogonal
+    alignment matrix ("M-step") is done via regular orthogonal procurstes.
+
+    Parameters
+    ----------
+        optimal_transport_lambda : float (default=0.1), optional
+            Regularization term of the Sinkhorn optimal transport algorithm.
+
+        optimal_transport_eps : float (default=0.01), optional
+            Tolerance parameter for the each Sinkhorn optimal transport
+            algorithm. I.e. tolerance for each "E-step".
+
+        optimal_transport_num_reps : int (default=1000), optional
+            Number of repetitions in each iteration of the iterative optimal
+            transport problem. I.e. maximum number of repetitions in each
+            "E-step".
+
+        iterative_eps : float (default=0), optional
+            Tolerance of the iterative optimal transport problem. I.e.
+            tolerance of the whole "EM" algorithm. Zero by default, which means
+            that the algorithm performs all `iterative_num_reps` number of
+            steps.
+
+        iterative_num_reps : int (default=100), optional
+            Number of reps in each iteration of the iterative optimal transport
+            problem. I.e. maxumum number of total iterations the whole "EM"
+            algorithm.
+
+        init : string, {"2d" (default), "sign_flips", "custom"}, optional
+
+            - "2d"
+                uses 2^d different initiazlizations, where d is the dimension.
+                specifically, uses all possible matrices with all entries real
+                and diagonal entries having magnitude 1 and 0s everywehre else.
+                for example, for d=2, tries [[1, 0], [0, 1]], [[1, 0], [0,
+                -1]], [[-1, 0], [0, 1]], and [[-1, 0], [0, -1]]. picks the best
+                one based on the value of the objective function.
+            - 'sign_flips'
+                for the first dimension, if two datasets have medians with
+                varying signs, flips all signs along this dimension for the
+                first dataset. then initializes to an identity.
+            - "custom"
+                expects either an initial matrix Q or initial matrix P during
+                the use of fit or fit_transform. uses initial Q provided,
+                unless it is not provided. if not provided - uses initial P. if
+                neither is given initializes to Q = I.
+
+        initial_Q : np.ndarray, shape (d, d) or None, optional (default=None)
+            An initial guess for the alignment matrix, if such exists. Ignored
+            if init alignment is set to anything other than 'custom'.
+            If None - initializes using an initial guess for P.
+            If None, and P is also None - initializes Q to identity matrix.
+
+        initial_P : np.ndarray, shape (n, m) or None, optional (default=None)
+            Initial guess for the initial transport matrix.
+            Only matters if Q=None.
```

Comment:
how about throw an error if user provides both instead?

Reply:
gr8 idea; will do that instead.

Reply:
also will make sure that init is set to custom.

Reply:
fxd i believe?